app/src/main/java/org/jellyfin/mobile/ui/screens/connect/ServerSuggestionPreview.kt

```diff
@@ -0,0 +1,96 @@
+package org.jellyfin.mobile.ui.screens.connect
```

Comment:
I feel like the amount of placeholder data this file contains just to show some previews in an IDE is too much and I'd rather not have previews then.

Reply:
I don't think the benefits of previewing entire screens outweigh the amount of mock data code needed in this case.